src/main/kotlin/io/meshcloud/dockerosb/persistence/ScheduledPushHandler.kt

```diff
@@ -0,0 +1,16 @@
+package io.meshcloud.dockerosb.persistence
+
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component
+class ScheduledPushHandler(val gitAccess: SynchronizedGitHandler) {
+
+  @Scheduled(
+    initialDelayString = "\${scheduling.push.initial-delay}",
+    fixedDelayString = "\${scheduling.push.pause-delay}"
```

Comment:
d: I wanted to ask again shortly about the jmeter suite. A comparison between the new and the old implementation would still be interesting. What happens when you run against the master branch (when adding the git lock fix to the master branch)? I guess it eventually fails, right? We should have that documeneted.

Reply:
f: And we should have the jmeter suite checked in under /src/jmeter or src/performanceTest or similar

Reply:
I have the jmeter suite attached to the notion page. But i will check it in also.
I can run it against master again as well and report the results to the notion pages as well.